Gemfile.lock

```diff
@@ -93,12 +94,14 @@ GEM
       rb-fsevent (>= 0.9.3)
       rb-inotify (>= 0.9.7)
     mercenary (0.3.5)
-    mini_portile2 (2.0.0)
+    mini_portile2 (2.4.0)
     minitest (5.8.4)
     multipart-post (2.0.0)
     net-dns (0.8.0)
-    nokogiri (1.6.7.2-x64-mingw32)
-      mini_portile2 (~> 2.0.0.rc2)
+    nokogiri (1.10.4)
+      mini_portile2 (~> 2.4.0)
+    nokogiri (1.10.4-x64-mingw32)
+      mini_portile2 (~> 2.4.0)
```

Comment:
I'm likewise not so sure about the -x64-mingw32 version of this.
I wonder if they're appearing because I originally developed site on my windows machine. It's possible they can just go away.